server/src/cache/pool.rs

```diff
@@ -59,29 +59,27 @@ impl RedisPool {
     pub async fn new(config: &crate::config::Config, key_folder: &str) -> Result<Self, ()> {
         // the .env file must have REDIS__URL="redis://\<host\>:\<port\>"
         let cfg: deadpool_redis::Config = config.redis.clone();
-        let Some(details) = cfg.url.clone() else {
+        let details = cfg.url.clone().ok_or_else(|| {
             cache_error!("(RedisPool new) no connection address found.");
```

Comment:
Remove `(RedisPool new)` prefixes from debug messages (4x). The debug output will already provide all the information needed to determine the origin of the message.

Reply:
568a2a762168b9f1fe611860989da50cc02c494a